zebra/zapi_msg.c

```diff
@@ -1895,6 +1895,8 @@ static int zapi_nhg_decode(struct stream *s, int cmd, struct zapi_nhg *api_nhg)
 	STREAM_GETL(s, api_nhg->resilience.idle_timer);
 	STREAM_GETL(s, api_nhg->resilience.unbalanced_timer);
 
+	STREAM_GETC(s, api_nhg->flags);
```

Comment:
Just a dumb question: is this compatible with older versions (if this new `flags` is not supported)?

Reply:
If you mean to increment zapi version , maybe we should.
Other than that, when using the same version, I don't see the point.
do you know the procedure to do this?